libs/vm/include/vm/object.hpp

```diff
@@ -87,6 +88,17 @@ struct IsAddress<T, typename std::enable_if_t<std::is_base_of<Address, T>::value
 {
 };
 
+template <typename T, typename = void>
+struct IsString : std::false_type
+{
+};
+template <typename T>
+struct IsString<
+    T, typename std::enable_if_t<std::is_base_of<String, typename std::decay<T>::type>::value>>
```

Comment:
`is_base_of_v`, `decay_t`?

Reply:
This is all over this file and other files of VM library.
I'm having another branch addressing this and many other issue of this kind.

Reply:
Addressing what? Is there an issue here?

Reply:
No, there is no an issue per say, I used _issues of this kind_ only to refer **verbosity** kind of issues pointed out by the comment on top which which triggered this discussion - instead of using unnecesarily verbose form of meta traits like `std::is_base_of<...>::value` and `typename std::decay<...>::type`, we can use less versose  `std::is_base_of_v<...>` and `std::decay_t<...>`,
where in the first case dropped `::value` and in the second case we dropped even more - unnecessary `typename` and `::type`